source/documentation/how-to/ha-vms.html.md

```diff
@@ -0,0 +1,86 @@
+
+The following resume behaviors are defined:
+
+- `AUTO_RESUME`: The VM is resumed automatically, as soon as the storage gets working again.  This is the simplest scenario – the VM simply pauses for the time of storage problems and then continues running again, without user intervention.  It is supposed to be used for non-highly available VMs without special requirements and for highly available VMs when using `KILL` behavior is not desirable for some reason.
+- `LEAVE_PAUSED`: The VM is never resumed automatically.  A user must handle the VM (run it or power it off) manually.  This behavior is suitable for VMs that require additional manual interventions after being paused, e.g. some remedy in their guest OS or software running there.
+- `KILL`: This is similar to `AUTO_RESUME` unless the VM is paused for too long.  If the storage problem is remedied within a given time interval (80 seconds by default), the VM is automatically resumed.  But if the storage problem persists for longer time, the VM is going to be killed sooner or later.  This behavior is suitable for highly available VMs, in order to permit them to start on another host, which doesn't suffer from the storage problem.
+
+Resume behavior can be selected on any VM, not just highly available VMs.  But there are special considerations regarding highly available VMs:
+
+- As long as a VM is paused, it may be prevented from starting on another, working, host.
+- Or a paused VM may be started on another host and later resumed on the original host, resulting in split brain (the VM runs in multiple instances at the same time).
+
+Note that a VM can be killed under some circumstances even when `AUTO_RESUME` or `LEAVE_PAUSED` policies are set if VM leases are used.
+
+## VM leases
+
+VM lease is a feature that guarantees a VM can't be run in multiple instances at the same time.  That guarantee makes restarting highly available VMs safer and thus easier.  Split brain is not possible when a VM has been started with a lease and the lease is kept enabled for the VM.
+
+VM leases can be enabled only for highly available VMs, in High Availability section of the VM editing dialog again.  Just select *Target Storage Domain for VM Lease* in the dialog.
+
+VM leases are thoroughly described in a separate [document about leases](/develop/release-management/features/storage/vm-leases/).
+
+To prevent split brain it's best to use VM leases on highly available VMs and set `KILL` resume behavior for them.  The same also helps restarting highly available VMs on other hosts in case of prolonged storage problems on some of the hosts.  The primary purpose of `KILL` resume behavior is to destroy a highly available VM before Engine moves the VM to Unknown status and attempts to start it elsewhere.  Of course, `KILL` behavior means killing the VM under the given conditions, so think about your setup before using it.
```

Comment:
"it's best to use VM leases on highly available VMs and set `KILL` resume behavior for them." -> you don't have an option here. If the VM is a HA VM with lease, it has to have the KILL behavior.